collect_app/src/main/java/org/odk/collect/android/preferences/AboutPreferencesFragment.java

```diff
@@ -48,13 +53,25 @@ public void onCreate(final Bundle savedInstanceState) {
 
 
     @Override
+    public void onStart() {
+        super.onStart();
+        mCustomTabHelper.bindCustomTabsService(this.getActivity(),uri);
+    }
+
+    @Override
     public boolean onPreferenceClick(Preference preference) {
         final String APP_PACKAGE_NAME = getActivity().getPackageName();
 
         switch (preference.getKey()) {
             case KEY_ODK_WEBSITE:
-                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(ODK_WEBSITE));
-                startActivity(intent);
+                if(mCustomTabHelper.getPackageName(getActivity()).size()!=0){
```

Comment:
okay. Actually I dint see any coding guidelines in wiki page. I will take care of that by now.